raiden-dapp/src/views/GeneralDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="general-screen">
+    <v-row class="general-screen__header" no-gutters>
+      <div class="general-screen__header__content">
+        <div class="general-screen__header__content__back">
+          <v-btn
+            height="40px"
+            text
+            icon
+            width="40px"
+            @click="onGeneralBackClicked()"
+          >
+            <v-img :src="require('../assets/back_arrow.svg')" max-width="34px">
+            </v-img>
+          </v-btn>
+        </div>
+        <div class="general-screen__header__content__title">
+          {{ $route.meta.title }}
+        </div>
+      </div>
+    </v-row>
+    <router-view></router-view>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import NavigationMixin from '@/mixins/navigation-mixin';
+
+@Component({})
+export default class GeneralDialog extends Mixins(NavigationMixin) {}
+</script>
+
+<style scoped lang="scss">
+@import '../scss/mixins';
+@import '../scss/fonts';
+
+.general-screen {
+  background-color: black; /* CHANGE */
+  border-radius: 10px;
+  height: 844px;
+  position: absolute;
+  width: 620px;
```

Comment:
Can't it be made with flexible height and width?

Reply:
It most likely can. I have some additional questions regarding this one, but perhaps it is easier to do it in a call.

Reply:
About this one. I would love to have it appear and disappear with some sort of sliding animation. That will most likely need changes to make it flexible. If you don't mind I suggest saving that until the next PR when I'm more clear about what to do?

Reply:
Ok cool